pallets/fund-admin/src/lib.rs

```diff
@@ -1132,6 +1439,8 @@ pub mod pallet {
 			let _ = <TransactionsInfo<T>>::clear(1000, None);
 			let _ = <TransactionsByDrawdown<T>>::clear(1000, None);
 
+			//TODO: Remove Revenues storages
```

Comment:
we need to remove the storage maps for revenues in order to push to prod (just to have a way to clean those)